background.js

```diff
@@ -1,4 +1,15 @@
-/* eslint-disable no-undef */
+const ION_SIGNUP_URL = "https://mozilla-ion.github.io/ion-core-addon/";
+const CORE_ADDON_ID = "ion-core-addon@mozilla.org";
```

Comment:
```suggestion
const CORE_ADDON_FIREFOX_ID = "ion-core-addon@mozilla.org";
```

Note that this addon id is only valid in Firefox: Chromium-based browser requires to use certificate and self-signing (or proper signing from the extension store) to have a stable id. At least, that's what I understood!

Reply:
Yep, that's right... I've been waiting to cross that bridge until we come to it, I am thinking we should be careful in how we submit it so it doesn't get blocked or anything.